hvac/api/secrets_engines/ad.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Active Directory methods module."""
+import json
+import logging
+
+from hvac import exceptions
+from hvac.api.vault_api_base import VaultApiBase
+
+DEFAULT_MOUNT_POINT = 'ad'
+
+
+class Ad(VaultApiBase):
+    """Google Cloud Secrets Engine (API).
+
+    Reference: https://www.vaultproject.io/api/secret/ad/index.html
+    """
+
+    def configure(self, binddn="", bindpass="", url="", userdn==None, upndomain==None, ttl=0, max_ttl=0,
```

Comment:
Have one too many `=`'s in a couple spots:

`userdn==None, upndomain==None,` -> `userdn=None, upndomain=None,`